documentation/resource_automatic_software_updates.md

```diff
@@ -0,0 +1,79 @@
+automatic_software_updates
+=====
+
+Use the **automatic_software_updates** resource to manage software update preferences.
```

Comment:
This should say manage the Automatic Software Update preferences. Also, let's get an inline screenshot of what this looks like in System Preferences. Here's the screenshot for you:

![image](https://user-images.githubusercontent.com/8458267/53369048-20d29b80-38ff-11e9-9e1c-ab0eb1a5d72c.png)